TowerDefenseGame/Assets/Tests/PlayerBaseTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.TestTools;
+using UnityEngine.UI;
+
+namespace Tests
+{
+    public class PlayerBaseTests
+    {
+        private PlayerBase playerBase;
+        private Text healthText;
+
+        [SetUp]
+        public void Setup()
+        {
+            var prefab = AssetDatabase.LoadAssetAtPath("Assets/Prefabs/PlayerBase.prefab", typeof(PlayerBase));
+            var playerBaseGameObject = MonoBehaviour.Instantiate(prefab) as PlayerBase;
+            playerBase = playerBaseGameObject.GetComponent<PlayerBase>();
+            healthText = playerBase.GetComponentInChildren<Text>();
+        }
+
+        [UnityTest]
+        public IEnumerator PlayerBaseLosesHealthWhenTakingDamage()
```

Comment:
This will pass, but as I mentioned in the playerbase.cs, if your defense is anything other than 0, then this test will fail.

Reply:
ya, I'll adjust the defense equation and add a test when I finish the testrunner stuff.